src/main/java/org/folio/circulationbff/service/impl/CheckInServiceImpl.java

```diff
@@ -246,6 +246,19 @@ private void rebuildCheckInStaffSlipContext(CheckInResponse response, SearchInst
       .effectiveLocationLibrary(fetchLocationLibraryName(location.getLibraryId()))
       .effectiveLocationSpecific(location.getName());
 
+    if(item.getItemLevelCallNumber() == null) {
```

Comment:
vice versa? != null?
```suggestion
    if (item.getItemLevelCallNumber() != null) {
```